content/tutorials/manage-the-rippled-server/installation/update-rippled-manually-on-centos-rhel.md

```diff
@@ -8,6 +8,18 @@ These instructions assume you have already [installed `rippled` from the `yum` r
 
 To update manually, complete the following steps:
 
+1. If you are upgrading to `rippled` 1.7.0 from an earlier version, re-add the the repository to get Ripple's updated GPG key. Otherwise, skip this step:
```

Comment:
```suggestion
1. If you are upgrading to `rippled` 1.7.0 from an earlier version, re-add the repository to get Ripple's updated GPG key. Otherwise, skip this step:
```